modules/@apostrophecms/command-menu/ui/apos/components/TheAposCommandMenu.vue

```diff
@@ -38,7 +38,7 @@ export default {
         modals
           .flatMap(group => {
             return Object.values(group.commands)
-              .filter(command => command.shortcut)
+              .filter(command => command.shortcut && command.listener !== false)
```

Comment:
@haroun lmk what you think about this strategy for manually placing your own listeners